src/dxtbx/boost_python/flumpy.cc

```diff
@@ -99,6 +101,31 @@ py::buffer_info get_buffer_specific(grid<scitbx::vec3<T>> flex) {
                          strides);
 }
 
+template <typename T>
+py::buffer_info get_buffer_specific(grid<cctbx::miller::index<T>> flex) {
+  std::vector<size_t> dim_sizes;
+  for (auto size : flex.accessor().all()) {
+    dim_sizes.push_back(size);
+  }
+  dim_sizes.push_back(3);
+
+  std::vector<size_t> strides;
+  for (int i = 0; i < dim_sizes.size(); ++i) {
+    auto stride = sizeof(T);
+    for (int j = i + 1; j < dim_sizes.size(); ++j) {
+      stride *= dim_sizes[j];
+    }
+    strides.push_back(stride);
+  }
+
+  return py::buffer_info(&flex.front(),
+                         sizeof(T),
+                         py::format_descriptor<T>::format(),
+                         dim_sizes.size(),
+                         dim_sizes,
+                         strides);
```

Comment:
Rather than duplicating, is there a way to get this to just call `get_buffer_specific<T>(grid<scitbx::vec3<T>> flex)`?

Reply:
Discussed this and: There is a way, but it isn't super clean because of incomplete inheritance. Just accept the duplication for now.